cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -270,3 +305,19 @@ def wait_for_operation(self, operation, region=None, zone=None):
     def parse_url(self, url):
         out = self._compute_resources.parse_url(url)
         return out if out else self._storage_resources.parse_url(url)
+
+    def get_resource(self, resource, identifier):
```

Comment:
`get_resource_by_name` would be more clear. Also `identifier` -> `name`.

Reply:
As discussed in the meeting, I changed the parameter name identifier to url_or_name to make it more clear.